### ISSUE TITLE
About Training Batch Size

### ISSUE THREAD

**Issue by QAQdev** [ByteDance-Seed/Bagel#135]
I have reviewed the training code, but I do not seem to have found a parameter that explicitly sets the training batch size. Instead, I find:

https://github.com/ByteDance-Seed/Bagel/blob/2f7d107c366a0a6befc1de2376bd37af0a0ccc58/train/pretrain_unified_navit.py#L562-L571

Could someone explain whether a dynamic batch size was used during training? Thanks in advance!

**Comment by Andy1621** [comment]
Hi, we use packed sequence for efficient training with native resolution.

**Comment by QAQdev** [comment]
Thanks! So how can I control the packed seq to fully utilize my GPU (80G)?

**Comment by Andy1621** [comment]
You can adjust the sequence length, condition drop ratio, resolution, sharding stragetegy and so on~

**Comment by QAQdev** [comment]
Thanks! I will try it.
